ui/preferences/src/main/res/xml/preferences_user_interface.xml

```diff
@@ -86,5 +89,9 @@
                 android:key="prefSwipe"
                 android:summary="@string/swipeactions_summary"
                 android:title="@string/swipeactions_label"/>
+        <Preference
+            android:key="prefHiddenDrawerItems"
+            android:summary="@string/pref_nav_drawer_items_sum"
```

Comment:
I would move this up, below the default page setting. That's a bit more closely related than the swipe actions

Reply:
Reworked